test/erqwest_SUITE.erl

```diff
@@ -200,6 +210,23 @@ proxy_basic_auth(Config) ->
                                                 }}]}),
   {ok, #{status := 200}} = erqwest:get(C1, <<"https://httpbin.org/get">>).
 
+cookies_enabled(_Config) ->
+  C0 = erqwest:make_client(#{cookie_store => true,
+                             follow_redirects => true}),
```

Comment:
I made follow redirects the default in 18a0c1a, since I thought it would be least surprising to keep `reqwest`'s defaults. Going forward I'll use a PR process for changes like this for better visibility!

Reply:
Ah I missed that. Cool. Updated in https://github.com/dlesl/erqwest/pull/7/commits/14709e92a8ba2054d6cbf663e959dd6e01feab53